### DISCUSSION TITLE
MAM Channels

### DISCUSSION THREAD

**Discussion by adam-635** [iota-button/iota-button#13] [Ideas]
Further utilize IOTA Streams (https://github.com/iotaledger/streams) to deliver functions such as:
- buyer / seller communication (pre/post purchase)
- delivery of digital assets
- embedding additional widgets within the conversation between buyer/seller (location on the map, link, etc.)  

[https://github.com/iotaledger/identity.ts] (DID) could be considered for this as well.

**Comment by sascha1337** [comment]
Hey, can we edit the name away from MAM Channels to IOTA Streams / Stream Channels ?
MAM is referencing the the legacy code | protocol, from the wording.

Thanks, and i love this repo, it just makes sense, great use case. 
Will PR in some time here and there. 🚀

**Comment by adam-635** [comment]
thanks @sascha1337 ! I've updated it now.

I hope I'll find some time in the future to do some work as well.
